src/main/cpp/util/AliasUtil.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * AliasUtil.h
+ *
+ *  Created on: Mar 12, 2017
+ *      Author: cedric
+ */
+
+#ifndef ALIASUTIL_H_
```

Comment:
Can you make the include guard identifier conform to Stride's format? That would be:

#ifndef ALIAS_UTIL_H_INCLUDED
#define ALIAS_UTIL_H_INCLUDED

...
#endif